basic-features.md

```diff
@@ -1,80 +1,136 @@
+| [Follower reads](/follower-read.md)                                                                      | Y            | Y            | Y            | Y            | N            | N            |
+| [Read historical data (tidb_snapshot)](/read-historical-data.md)                                         | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Optimizer hints](/optimizer-hints.md)                                                                   | Y            | Y            | Y            | Y            | Y            | Y            |
+| [MPP Exection Engine](/explain-mpp.md)                                                                           | Y            | Y            | N            | N            | N            | N            |
+| **Data definition language (DDL)**                                                                       | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| Basic `CREATE`, `DROP`, `ALTER`, `RENAME`, `TRUNCATE`                                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Generated columns](/generated-columns.md)                                                               | Experimental | Experimental | Experimental | Experimental | Experimental | Experimental |
+| [Views](/views.md)                                                                                       | Y            | Y            | Y            | Y            | Y            | N            |
+| [Sequences](/sql-statements/sql-statement-create-sequence.md)                                            | Y            | Y            | Y            | N            | N            | N            |
+| [Auto increment](/auto-increment.md)                                                                     | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Auto random](/auto-random.md)                                                                           | Y            | Y            | Y            | N            | N            | N            |
+| [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md)                                 | Y            | Y            | Y            | N            | N            | N            |
+| Multi schema change: add column(s)                                                                       | Y            | Y            | N            | N            | N            | N            |
+| [Change column type](/sql-statements/sql-statement-modify-column.md)                                      | Y            | N            | N            | N            | N            | N            |
```

Comment:
@bb7133 Can you pleaser view this section (DDL)?

Reply:
Yes, LGTM.